tests/channel-rate-limit.test.ts

```
@@ -0,0 +1,488 @@
+import { randomUUID } from 'node:crypto'
+import { afterAll, beforeAll, describe, expect, it } from 'vitest'
+import { BASE_URL, getBaseData, PLUGIN_BASE_URL, resetAndSeedAppData, resetAppData, resetAppDataStats, headers } from './test-utils.ts'
```

ESLint

[error] Expected "headers" to come before "resetAppDataStats".

(perfectionist/sort-named-imports)